e2e/cypress/support/api/role.js

```diff
@@ -0,0 +1,384 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import xor from 'lodash.xor';
+
+// *****************************************************************************
+// Preferences
+// https://api.mattermost.com/#tag/roles
+// *****************************************************************************
+
+export const defaultRolesPermissions = {
+    channel_admin: [
+        'create_post',
+        'manage_private_channel_members',
+        'read_public_channel_groups',
+        'manage_channel_roles',
+        'use_channel_mentions',
+        'use_group_mentions',
+        'read_private_channel_groups',
+        'remove_reaction',
+        'add_reaction',
+        'manage_public_channel_members',
```

Comment:
@saturninoabril can we add `convert_public_channel_to_private` & `convert_private_channel_to_public` here (for team admin role) to match the server defaults

Reply:
It should also be added to `system_user_manager` & `system_manager` roles

Reply:
I initially had those but were eventually removed after I've got the same bad request when patching the roles. See [logs](https://circleci.com/api/v1.1/project/github/saturninoabril/mattermost-cypress-docker/917/output/110/0?file=true&allocation-id=5f5486c12cdb5f297b4294e5-0-build%2F3F1CF14A). I've seen your recent PR for it. Maybe I'll wait for another build on master and see if it fixes the issue.